src/index.ts

```diff
@@ -1,9 +1,14 @@
-import type { PiniaPluginContext } from 'pinia'
+import type { PiniaPluginContext, StateTree } from 'pinia'
 
 import pick from './pick'
 
 export type StorageLike = Pick<Storage, 'getItem' | 'setItem'>
 
+export interface Serializer {
```

Comment:
no need for an additional interface

Reply:
Resolved in https://github.com/prazdevs/pinia-plugin-persistedstate/pull/22#issuecomment-1056880090